doc/manual/source/man/dnst-signzone.rst

```diff
@@ -0,0 +1,108 @@
+dnst-signzone
+===============
+
+Synopsis
+--------
+
+:program:`dnst signzone` [``OPTIONS``] ``<ZONEFILE>`` ``<KEY>...``
+
+Description
+-----------
+
+**dnst signzone** signs the zone with the given key(s).
+
+Keys must be specified by their base name (usually ``K<name>+<alg>+<id>``),
+i.e. WITHOUT the ``.private`` or ``.key`` extension. Both ``.private`` and
+``.key`` files are required.
+
+A date can be a timestamp (seconds since the epoch), or of the form
+<YYYYMMdd[hhmmss]>.
+
+
+Options
+-------
+
+.. option:: <ZONEFILE>
+
+      The zonefile to sign.
+
+.. option:: <KEY>...
+
+      The keys to sign the zone with.
+
+.. option:: -b
+
+      Use a more readable layout in the signed zone file and print comments on
```

Comment:
The layout of `dnst` output is NOT affected by this option, it only adds comments. The original `ldns-signzone` sets an internal flag `LDNS_COMMENT_LAYOUT` which has an effect in various places in the ldns library code on adding yet more comments (e.g. `; <void>`, `;; Empty non-terminal: <name>` and `;; Zone: <zone name>`) and adds some line breaks, but these effects have not been completely replicated in the `dnst` version yet.

Reply:
Further, per the original `ldns-signzone` man page: "Without this option, only DNSKEY RR's will have their Key Tag annotated in the comment text." - this IS implemented, but reads from the current `dnst` man page text as if no comments are ouput unless `-b` is used which is not true.